Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeFootIKNode.cpp

```diff
@@ -311,18 +311,20 @@ namespace EMotionFX
             heightOffset += uniqueData->m_legs[legId].m_toeHeight * actorInstanceScale;
         }
 
-        Integration::RaycastRequests::RaycastRequest rayRequest;
+        Integration::IRaycastRequests::RaycastRequest rayRequest;
```

Comment:
question: Is it absolutely necessary to add the `I` prefix to this type? If this is the convention for interfaces so be it but this is one of those Hungarian notation style things that may be redundant/not advised these days

Reply:
Especially given that `rayRequest` looks like an instantiation of a concrete class in this declaration, not really an interface